var/spack/repos/builtin/packages/g2/package.py

```diff
@@ -62,3 +64,8 @@ def setup_run_environment(self, env):
             lib = find_libraries("libg2_" + suffix, root=self.prefix, shared=False, recursive=True)
             env.set("G2_LIB" + suffix, lib[0])
             env.set("G2_INC" + suffix, join_path(self.prefix, "include_" + suffix))
+
+    def check(self):
```

Comment:
@AlexanderRichert-NOAA this is in our private version of the package file, should it be only there? Or should this check() function be added to the main spack repo as well?

Reply:
I've been including that in our other recipes, so I would say let's keep it here. In principle it should be just as useful to external developers who want to run unit tests.